packages/node/src/types.ts

```diff
@@ -167,16 +195,25 @@ export interface WithReadPowers {
   url?: never
 }
 
+/**
+ * Options having a `trustRoot` property
+ */
 export interface WithTrustRoot {
   trustRoot?: boolean
 }
 
+/**
+ * Options for `scuttleGlobalThis`
+ */
 export interface ScuttleGlobalThisOptions {
   enabled?: boolean
   exceptions?: string[]
   scuttlerName?: string
 }
```

Comment:
(must) this is wrong. Import the type from core. exception are `Array<string|RegExp>`